setup.py

```diff
@@ -60,21 +60,8 @@ def run(self):
         "docs": ("sphinx", "pallets-sphinx-themes"),
         "lint": ("pre-commit ~= 1.17"),
         "jwt": ("PyJWT >= 1.4.0", "cryptography >= 2.0.0"),
-        "tests": (
-            "coverage",
-            "flake8",
-            "flake8-config-4catalyzer",
-            "psycopg2-binary",
-            "pytest",
-        ),
-    },
-    cmdclass={
-        "clean": system("rm -rf build dist *.egg-info"),
-        "docs": system("make -C docs html"),
-        "package": system("python setup.py sdist bdist_wheel"),
-        "publish": system("twine upload dist/*"),
-        "release": system("python setup.py clean package publish"),
-        "test": system("tox"),
+        "tests": ("coverage", "flake8", "psycopg2-binary", "pytest"),
     },
+    cmdclass={"test": system("tox")},
```

Comment:
We can probably get rid of this and the `system` function altogether in favor of using `tox` directly. This is what we do in all the marshmallow libs (https://github.com/marshmallow-code/marshmallow/pull/1025).

I can do that in a follow-up PR.